axelor-stock/src/main/resources/views/LogisticalForm.xml

```diff
@@ -21,7 +21,7 @@
     </grid>
 
     <form model="com.axelor.apps.stock.db.LogisticalForm" title="Logistical form" name="logistical-form-form"
-        width="large" onNew="action-logistical-form-new" onSave="action-logistical-form-save">
+        width="large" onNew="action-logistical-form-new" onSave="action-logistical-form-save,action-logistical-form-validate-mass-on-save">
         <toolbar>
```

Comment:
action-logistical-form-save is already a action-group, add this new action in that action-group